apps/native-component-list/src/screens/Image/tests/methods.tsx

```diff
@@ -0,0 +1,19 @@
+import { images } from '../images';
+import { ImageMethodNames, ImageTestGroup } from '../types';
+
+const imageTests: ImageTestGroup = {
+  name: 'Methods',
+  tests: [
+    {
+      name: 'prefetch',
+      props: {
+        source: images.uri_png,
+        defaultSource: images.require_monochrome,
+      },
+      loadOnDemand: true,
+      method: ImageMethodNames.Prefetch,
```

Comment:
```suggestion
      method: <ExpoImage.methodName>,
```
What about passing a function here? Then in `ImageMethods.tsx -> getImageMethodResult` we could take advantage of [Rest params](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) to receive arguments and [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to pass them into callee.
I believe this would allow us to resign from determining what method is being tested w/o wall of if statements.